dracut/driver_updates.py

```diff
@@ -491,13 +492,25 @@ def _process_driver_disk(dev, interactive=False):
 
     return modules
 
-def process_driver_rpm(rpm):
+def process_driver_rpm(rpm, dev = None):
```

Comment:
I'd just leave out the spaces in default argument for consistency (dev=None).
Sorry I don't have any better remark to the patch, looks good to me.